webcompat/static/js/lib/bugform.js

```diff
@@ -18,6 +18,7 @@ function BugForm() {
   this.submitTypeInput = $("#submit_type:hidden");
   this.uploadLabel = $(".js-label-upload");
   this.uploadLoader = $(".js-image-loader");
+  this.urlParamRegExp = /url=([^&]*)/;
```

Comment:
I'd possibly check for `/url=([^&]+)/` and immediately ignore empty values for `url=`.

Reply:
Good suggestion. Possibly worth a follow up bug and patch, but if it's not too much work let's do it.